api/api.go

```diff
@@ -180,14 +184,40 @@ func (api *API) status(w http.ResponseWriter, req *http.Request) {
 			"buildDate": version.BuildDate,
 			"goVersion": version.GoVersion,
 		},
-		Uptime: api.uptime,
+		Uptime:     api.uptime,
+		MeshStatus: getStrippedDownMeshStatus(api),
 	}
 
 	api.mtx.RUnlock()
 
 	respond(w, status)
 }
 
+type strippedDownMeshStatus struct {
+	Connections []strippedDownLocalConnectionStatus `json:"connections"`
+}
+
+type strippedDownLocalConnectionStatus struct {
+	Address string `json:"address"`
+	State   string `json:"state"`
+}
+
+func getStrippedDownMeshStatus(api *API) strippedDownMeshStatus {
+	status := mesh.NewStatus(api.mrouter)
+	strippedStatus := strippedDownMeshStatus{
```

Comment:
Technically same here ~~stripped~~, but I can see that that will cause collisions, generally speaking though you can keep the variable names much shorter. For example:

```go
s := mesh.NewStatus(api.mrouter)
status := meshStatus{
...
}
```